torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1,14 +1,84 @@
+#include <stack>
 #include <torch/csrc/jit/passes/quantization.h>
 
 #include <torch/csrc/jit/ir.h>
 #include <torch/csrc/jit/node_hashing.h>
 #include <torch/csrc/jit/passes/alias_analysis.h>
 
-#include <stack>
+#include <unordered_set>
 
 namespace torch {
 namespace jit {
+namespace {
+// QuantizerUtils
+
+int checkIfNodeValuesQuantizable(Node* n) {
+  // This is map for quantizable nodes. It will be expanded in future to
+  // support more ops and patterns.
+  static const std::unordered_set<std::string> nodeLookup =
```

Comment:
Please use the OperatorSet object and not operator names:

```
OperatorSet cannot_propagate_shape_by_running_it = {
      "aten::solve(Tensor self, Tensor A) -> (Tensor, Tensor)",
      "aten::inverse(Tensor self) -> Tensor",
  };
```

This enforces that when someone changes the arguments to an operator, they also must look at the places where it is matched.